peps/pep-0789.rst

```diff
@@ -0,0 +1,468 @@
+PEP: 789
+Title: Preventing task-cancellation bugs by limiting yield in async generators
+Author: Zac Hatfield-Dodds <zac@zhd.dev>,
+        Nathaniel J. Smith <njs@pobox.com>
+PEP-Delegate:
+Discussions-To: https://discuss.python.org/t/preventing-yield-inside-certain-context-managers/1091
```

Comment:
This link should go into the Post-History thread, but a new thread should be opened after the PEP is merged (you can add it in a followup PR).